packages/newsroom-signup/src/NewsroomProfile/LearnMoreButton.tsx

```diff
@@ -78,45 +78,78 @@ export class LearnMoreButton extends React.Component<LearnMoreButtonProps, Learn
+              few key terms. In this section we'll take you through the steps to join the Civil network. But first,
+              let’s start with a few key terms.
             </Paragraph>
+
+            <SectionHeader>What is the Civil Registry</SectionHeader>
             <Paragraph>
-              When creating your Newsroom Profile, we will ask you to complete the application asking for information
-              about your newsroom. We ask that any newsroom applying to the registry provide the information to the best
-              of their ability.
+              The Civil Registry is the online destination where prospective Civil Newsrooms apply to be part of the
+              network.
             </Paragraph>
-            <SectionHeader>What is the Civil Registry</SectionHeader>
             <Paragraph>
-              Civil Registry is the destination to which prospective Civil Newsrooms must apply to access publishing
-              rights on the Civil platform. You can apply to the Civil Registry after creating your Newsroom Smart
-              Contract.
+              Newsrooms on the Civil Registry have been vetted by the community and deemed it as having a credible,
+              journalistic practices as outlined in the{" "}
+              <a href="https://civil.co/constitution/" target="_blank">
+                Civil Constitution
+              </a>.
             </Paragraph>
+
+            <SectionHeader>Civil Tokens</SectionHeader>
             <Paragraph>
-              Being included in the Civil Registry means that the community has vetted a Newsroom and deemed it as
-              having a credible, journalistic mission, and that it has pledged to adhere to the journalistic ethics
-              outlined by the Civil Constitution.
+              Civil tokens (CVL) enable the "community-owned and operated" Civil model to work. A token is a piece of
+              software, it’s not a physical coin. But it does have value and represents a piece of ownership in the
+              network.
             </Paragraph>
+            <Paragraph>
+              For those that want to get more technical, CVL is a utility token based on the ERC20 protocol. It’s a
+              value stored in a decentralized database that’s managed by Civil’s smart contracts, which allows the Civil
+              platform to interact with the Civil token. CVL is the software that bridges Civil with the Ethereum
+              blockchain.
+            </Paragraph>
+
+            <SectionHeader>Ether (ETH)</SectionHeader>
+            <Paragraph>
+              Ether (ETH) is different cryptocurrency than CVL but you will need some to to pay for certain actions on
+              Civil like sending tokens. That takes a small amount of computing effort, called Gas, which must be paid
+              for in ETH. When sending a transaction, you set a gas limit. You must set this limit carefully, because if
```

Comment:
probably shouldn't mention this as they literally never have to choose their own gas limit

Reply:
okay, lets remove it.

Reply:
@walfly here's the updated copy 

`Ether (ETH) is the cryptocurrency or fuel for the Ethereum blockchain. You will need ETH to pay for certain transcations on Civil. Each transaction takes a small amount of computing effort, called Gas, which is paid for in ETH. ` The paragraph felt clunky.

Reply:
I like it